src/main/java/com/ptudw/web/service/UserService.java

```diff
@@ -365,4 +365,31 @@ private void clearUserCaches(User user) {
             Objects.requireNonNull(cacheManager.getCache(UserRepository.USERS_BY_EMAIL_CACHE)).evict(user.getEmail());
         }
     }
+
+    public void mapStudentIdsByExcel(List<List<String>> data) {
+        Authority studentAuthority = new Authority();
+        studentAuthority.setName(AuthoritiesConstants.STUDENT);
+
+        List<User> usersToUpdate = new ArrayList<>();
+
+        Optional
+            .ofNullable(data)
+            .orElse(Collections.emptyList())
+            .stream()
+            .filter(Objects::nonNull)
+            .forEach(row -> {
+                if (row.size() == 2) {
```

Comment:
why row.size() = 2 ?. please define a constant for it

Reply:
I fixed